src/settingsvpnmodel.cpp

```diff
@@ -838,6 +838,10 @@ QVariantMap SettingsVpnModel::processOpenVpnProvisioningFile(QFile &provisioning
                      if (!arguments.isEmpty()) {
                          rv.insert(QStringLiteral("OpenVPN.BlockIPv6"), arguments.join(QChar(' ')));
                      }
+                } else if (directive == QStringLiteral("up") || directive == QStringLiteral("down")) {
+                    // Ignore both up and down scripts as they would interfere with ConnMan and
+                    // we do not ship any OpenVPN scripts with the package.
+                    qInfo() << "Ignoring " << directive << " script");
```

Comment:
Does it compile?

Reply:
Also, by default `<<` inserts a space, there's no need to add more spaces, e.g. after "Ignoring" and before "script"

Reply:
I guess here we see a prime example of C -> C++ transformation. Halfway there!